### PR TITLE
[LICENSE] Reorganize rat-excludes file to ease license auditing

### PR DESCRIPTION
**Pull request by szha**
## Description ##
Reorganize rat-excludes file to ease license auditing

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [x] Reorganize rat-excludes file to ease license auditing

### CODE REVIEW

**Comment by szha** [comment]
cc @TristonC @sandeep-krishnamurthy

File: tools/apache_rat_license_check/rat-excludes [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+.*\.lds
+.*\.in
+.*\.diff
+.*\.edl
+.*\.md5
+.*\.csv
+.*\.log
+.*\.interp
+.*\.tokens
+.*\.cPickle
```
Comment (leezu) [inline tools/apache_rat_license_check/rat-excludes:79]: Do we need to keep these files in source control?

Comment (szha) [reply tools/apache_rat_license_check/rat-excludes:79]: I think we still have some of such data files in the code base and I'd like to limit the scope of the change.

**Review by samskalicky — COMMENTED** [review]
THANKS! (thats a big thanks ;-D) 

Any particular reason to move the file to `tools/apache_rat_license_check/rat-excludes` instead of just putting it at the top level of the repo?

**Comment by szha** [comment]
> Any particular reason to move the file to tools/apache_rat_license_check/rat-excludes instead of just putting it at the top level of the repo?

I kept the folder so that the `apache_rat_license_check/README.md` can be together with the allowlist file. Looking at the readme, I think it's ok to remove this file and incorporate some of the information into the allowlist as comments. I will go ahead and make the change.

**Review by samskalicky — APPROVED** [review]
Thanks @szha !

**Comment by szha** [comment]
I think the PR remained open long enough for feedback. I will merge it for now and I'm happy to update if there's any follow-up comment.

**Pull request merged**